### ISSUE TITLE
No media,please insert disk

### ISSUE THREAD

**Issue by 1558204459** [brandonlw/Psychson#129]
Why i can not open my disk after making it to be a badusb,it tips that "please insert disk"..........., can anyone help me?

**Comment by Bonjour123** [comment]
Yes, the mass storage feature should be preserved but I can't figure out why it isn't the case. If anyone knows, lease help ^^
